resource_customizations/numaplane.numaproj.io/ISBServiceRollout/actions/disable-force-promote/action.lua

```diff
@@ -0,0 +1,2 @@
+obj.spec.strategy.progressive.forcePromote = false
```

Comment:
Based on the nil checks below, seems like we might need nil checks for this assignment?

Reply:
I didn't add any nil checks here since the button is only enabled when we check that forcePromote is explicitly true in the spec. Would it be safer to also have those same nil checks here?

Reply:
Fair point... but yeah I'd go ahead and add it, just so there's no possibility of a race condition between the discovery script and the action script.